### PR TITLE
Feature/style skeletons

### PR DESCRIPTION
**Pull request by gjtrecny**
### What (if any) features are you implementing?
- Styling and layout throughout all the pages on the app
### What (if anything) did you refactor?
- Added some additional classes and tiles for possible future features
### Were there any issues that arose?
- Unsure of how to implement mix-ins and variables here since our SASS files are nested with their JS counterparts

### Refactor checklist
- [x] Check variable and parameter names: are they descriptive and unique?
- [ ] Is the code DRY? SRP?
- [x] Is it easy for others to read your code? (explanatory comments needed?)
- [x] Is the syntax consistent? (e.g. use of semicolons, blank lines)
- [x] Are commit messages formatted correctly?

### CODE REVIEW

File: src/Components/Delay.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import React, { useEffect, useState } from "react";
+
```
Comment (kavakai) [inline src/Components/Delay.js:2]: I think I see whats happening here, but would be awesome to go over this

File: src/Pages/LandingPage/LandingPage.js [hunk 2]
```diff
@@ -3,17 +3,23 @@ import { Transition } from 'react-transition-group';
...
     return (
         <section className='landing-page'>
-            <h1>{quote}</h1>
-            <Link to={'/features'}>
-                <button className='landing-button'>Get Mindfulness Through the Wire</button>
-            </Link>
+            <article className='open-quote fade-in-fwd'>
+                <h1>{quote}</h1>
+            </article>
+            <Delayed>
```
Comment (kavakai) [inline src/Pages/LandingPage/LandingPage.js:18]: I think the Delayed component is making a little more sense now.

**Review by kavakai — APPROVED** [review]
Awesome work on this Gabby.

File: public/index.html [hunk 3]
```diff
@@ -3,6 +3,9 @@
   <head>
     <meta charset="utf-8" />
     <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
+    <link rel="preconnect" href="https://fonts.googleapis.com">
```
Comment (anewb87) [inline public/index.html:6]: This font is sweet! But it didn't bring in the underline that it shows on google fonts. So boo. But still cool.

File: src/Components/Delay.js (hunk 1 above)
Comment (anewb87) [reply src/Components/Delay.js:2]: Yes, please!

File: src/SASS/fade-in.scss [hunk 4]
```diff
@@ -0,0 +1,28 @@
+.fade-in-fwd {
```
Comment (anewb87) [inline src/SASS/fade-in.scss:1]: Let the fun begin!

File: src/Pages/MoodPage/MoodPage.scss [hunk 5]
```diff
@@ -0,0 +1,13 @@
...
+    display: flex;
+    flex-direction: column;
+    justify-content: space-around;
+    align-items: center;
+    border: 5px black solid;
+    width: 100vw;
+    height: 100vh;
+    background-color: #f7e1e9;
+    font-family: 'Montserrat', sans-serif;
+    color: whitesmoke;
```
Comment (anewb87) [inline src/Pages/MoodPage/MoodPage.scss:11]: Whitesmoke for the win. Always.

**Review by anewb87 — COMMENTED** [review]
Looks great!

**Pull request merged**